dist/obs-server.spec

```diff
@@ -336,6 +336,16 @@ export DESTDIR=$RPM_BUILD_ROOT
 
 export OBS_VERSION="%{version}"
 DESTDIR=%{buildroot} make install FILLUPDIR=%{_fillupdir}
+
+%if 0%{?suse_version}
+systemd_services="$(basename --multiple --suffix .service %{buildroot}%{_unitdir}/*.service)"
+for systemd_service in $systemd_services; do
+    if [[ $systemd_service != *"@"* ]]; then
+        ln -sf /usr/sbin/service %{buildroot}%{_sbindir}/rc${systemd_service}
+    fi
+done
+%endif
+
```

Comment:
This makes absolute sense to me. It`s a distro specific functionalty and should go into specfile.
+1